packages/service-library/src/servicelib/fastapi/long_running_tasks/_context_manager.py

```diff
@@ -124,3 +126,11 @@ async def _wait_for_task_result() -> Any:
             timeout=task_timeout,
             exception=e,
         ) from e
+    except Exception as e:
+        error = TaskExceptionError(
+            task_id=task_id,
+            exception=e,
+            traceback=f"check remote side for logs, HINT: service replying to: '{client._base_url}' for '{task_id=}'",  # noqa: SLF001  # pylint:disable=protected-access
```

Comment:
what means `remote side` here? you are talking about the server? the client?
will this not now create too many same logs all in the logs?
why is it now a warning if it is an error? why then not use `_logger.exception` ?

You are using private fields here. 

So basically this is here the client to some server that is responding an error right?
maybe it would be easier to define a _repr_ in the client and use it?